contrib/inventory/infoblox.py

```diff
@@ -30,6 +30,7 @@
 
 
 CONFIG_FILES = [
+    os.environ.get('INFOBLOX_CONFIG_FILE', None),
```

Comment:
use the plugin config infrastructure instead, it will allow the user to set it multiple ways